experiments/amg2023/openmp/ramble.yaml

```diff
@@ -47,7 +47,11 @@ ramble:
   software:
     packages:
       hypre-omp:
-        pkg_spec: hypre@2.31.0 +mpi+openmp+mixedint~fortran{modifier_spack_variant}
+        # spack_spec: hypre@2.31.0 +mpi+openmp+mixedint~fortran{modifier_spack_variant}
```

Comment:
@gracenansamba It would be good to move the amg2023 changes to its own PR 
https://github.com/LLNL/benchpark/pull/318